smarsy/parse.py

```diff
@@ -121,3 +121,62 @@ def login():
                                     get_user_credentials(),
                                     get_headers())
     return response
+
+
+def bs_safeget(soup, *args):
+    """
+        Utility function used to get a content string from a
+        Beautiful Soup object and tuple of selectors. Returns False
+        if no object is found for the given selector
+        """
+    for arg in args:
+        selectedElems = soup.select(arg)
+    if selectedElems is not None and len(selectedElems) > 0:
+        return selectedElems[0]
+    return False
+
+
+def create_parents_dict(parent_html) -> dict:
+    """
+    Function receive HTML and return parent dictionary with given keys
+    """
+    parent_img = bs_safeget(parent_html, '[valign=top]', 'img[src]')
+    # parent_dict = {}
+    # parent_list = []
+    # parents_keys = ['parent_img', 'parent_name', 'parent_surname',
+    #                 'parent_middlename', 'parent_type',
+    #                 'parent_birth_date']
+    if parent_img:
+        if hasattr(parent_img, 'src'):
+            print('True')
```

Comment:
is it tested? I mean print statement, hasattr?

Reply:
`print('True')`
it's just a stub, could write pass. I made a print because I did not know what to do with the data. You said don't create an list. After I know what to do with the data, I can test hasattr

Reply:
I'm not sure what does hasattr supposed to do here. It's not working on my machine as well

Reply:
hasattr it's Built-in Python Function
[https://docs.python.org/3/library/functions.html#hasattr](url)

Reply:
I'm asking what is it used for in our case?